src/network/networkd-network.c

```diff
@@ -229,6 +229,19 @@ int network_verify(Network *network) {
         if (network->ignore_carrier_loss < 0)
                 network->ignore_carrier_loss = network->configure_without_carrier;
 
+        if (!IN_SET(network->activation_policy, ACTIVATION_POLICY_UP, ACTIVATION_POLICY_ALWAYS_UP)) {
```

Comment:
Maybe, it should be included in `ACTIVATION_POLICY_BOUND`?

Reply:
hmm, I wasn't really sure.

For 'down' and 'always-down', we should definitely default to not required.
For 'up' and 'always-up', we should definitely keep the existing default of required.

For 'bound', I wasn't sure, but defaulting to required seems ok.

For 'manual', should we default to required or not?

Reply:
pushed a new version that does this.